dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -31,14 +31,14 @@ def new_general
 
       @form_params = {
         environment: Rails.env,
-        user_id: current_user.id,
-        user_name: current_user.name,
-        user_email: current_user.email,
-        workshop_id: workshop.id,
-        workshop_course: workshop.course,
-        workshop_subject: workshop.subject,
-        regional_partner_name: workshop.regional_partner&.name,
-        session_id: session&.id,
+        userId: current_user.id,
+        userName: current_user.name,
+        userEmail: current_user.email,
+        workshopId: workshop.id,
+        workshopCourse: workshop.course,
+        workshopSubject: workshop.subject,
+        regionalPartnerName: workshop.regional_partner&.name,
+        sessionId: session&.id,
```

Comment:
Note: this new form has a param called `csp_or_csd`, not camelized, so I changed the embed helper (see below) to just stringify the keys but otherwise leave them alone. This will be more clear. The keys here will match those in JotForm.